### PR TITLE
fix 9032

### LINKED ISSUE #9030: Recipe collision on grinding ball from EnderIO
### Your GTNH Discord Username

bombcar#1042

### Your Pack Version

2.1.2.0pre4

### Your Server

SP

### Type of Server

Single Player

### Your Expectation

Don't collide

### The Reality

Collision

image

image

### Your Proposal

This appears to be on grinding balls made out of some GT++ alloys, but I didn't check all of them. 

Found using https://github.com/D-Cysteine/nei-custom-diagram/wiki/Diagrams#gregtech-recipe-debugger

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

### LINKED ISSUE #9032: make ender quarry and ender pumps placeable in world
### Your GTNH Discord Username

boubou_19#2706

### Your Pack Version

2.1.2.0-pre4

### Original Request

make ender quarry and ender pumps placeable in world

### Responding developer

@boubou19

### Context

https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/blob/71879199f271f1bad49e215186c7e6c80eacf687/config/GTNewHorizons/blocklimiter.cfg#L10-L14

### Developer's Promise

Change it so non OP players can place the EQ and EP in at least a dim.

### Final Checklist

- [X] The developer in question has instructed me to create such an issue.
- [X] I believe the developer is not causally joking around.
- [X] I understand this reminder might take a long time to implement no matter what the developer has promised. (e.g. unforeseeable life difficulty, computer breakdowns, forgetting, laziness, etc)
- [X] I understand other people (developers or not) might not like this feature and would oppose its integration.
- [X] I believe this feature would make the pack better.

### PR DESCRIPTION
**Pull request by bombcar**
Fix #9032 and fix #9030

### CODE REVIEW

**Comment by bombcar** [comment]
sets default config to officials, blocking in overworked

File: config/GTNewHorizons/blocklimiter.cfg [hunk 1]
```diff
@@ -7,10 +7,10 @@ debug {
...
 
 main {
-    # Define your Blocks here. Syntax is: [modID]:[BlockID];[DimID];... if you don't add a Dimension (e.g. minecraft:dirt) it will be denied in every dimension [default: ]
+	# Define your Blocks here. Syntax is: [modID]:[BlockID];[DimID];... if you don't add a Dimension (e.g. minecraft:dirt) it will be denied in every dimension [default: ]
     S:BlockList <
-        ExtraUtilities:enderThermicPump;0;
-        ExtraUtilities:enderQuarry;0;
+        ExtraUtilities:enderThermicPump;0;0;1;
+        ExtraUtilities:enderQuarry;0;0;1;
      >
```
Comment (boubou19) [inline config/GTNewHorizons/blocklimiter.cfg:14]: those two should be removed from the config, and server owners should put what they need here. There is no reason to block those in OW and in End if players are on SP

**Review by boubou19 — CHANGES_REQUESTED** [review]

**Comment by Dream-Master** [comment]
@bombcar do me a favour and not mix stuff inside one pr. Thanks

**Review by Dream-Master — CHANGES_REQUESTED** [review]
not mix stuff together. Do 2 pr`s thanks

File: config/GTNewHorizons/blocklimiter.cfg (hunk 1 above)
Comment (bombcar) [reply config/GTNewHorizons/blocklimiter.cfg:14]: so comment it out entirely and let server owners find it?

I assume in general we tune the pack for "Single player" and assume server operators know what to look for - do we have a "gotchas on upgrade" somewhere?

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**